scalableminds/webknossos
# Upgrade documentation package to fix broken github links

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- I ran `yarn docs` locally and opened `webknossos/public/docs/frontend-api/index.html` before the change and after the update to confirm that the links are indeed fixed by the update

### Issues:
- fixes broken links in https://webknossos.org/assets/docs/frontend-api/index.html

------
(Please delete unneded items, merge only when none are left open)
